spec/ruby/core/file/atime_spec.rb

```diff
@@ -22,6 +22,11 @@
       if supports_subseconds != 0
         expected_time = Time.at(Time.now.to_i + 0.123456)
         File.utime expected_time, 0, @file
+        # FIXME: A random failing test on Travis ppc64le.
+        # https://bugs.ruby-lang.org/issues/17926
+        if ENV.key?('TRAVIS') && ENV['TRAVIS_CPU_ARCH'] == 'ppc64le'
+          skip '[ruby-core:17926] A random failure on Travis ppc64le'
+        end
```

Comment:
It would be better to use `platform_is_not` below the  `platform_is :linux, :windows do` above.
I guess it's an issue of linux-ppc64le (utimes(2) seems to ignore microseconds), not something specific to TravisCI.

Reply:
I did that in https://github.com/ruby/spec/commit/2168fda0cd22ce6d62377b9491e86fc6914e1006

Reply:
@eregon oh thanks! But I have observed this failure on only Travis' ppc64le.  I have not observed it on RubyCI's "ppc64le (Ubuntu) on <https://rubyci.org/>. I wanted to show the situation.

Reply:
I did not want to hide the failure unnecessarily on other environments.

Reply:
I see. IMHO it doesn't matter too much, it's OK to `platform_is_not` when some subset of the runs don't work, especially for exotic platforms. Probably it's something to do with the OS/kernel version in this case, and that's probably too much trouble to guard to be worth it.